modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClosureAngularClientCodegen.java

```diff
@@ -64,15 +70,11 @@ public JavascriptClosureAngularClientCodegen() {
 
         typeMapping.put("binary", "string");
 
-        outputFolder = "generated-code/javascript-closure-angular";
-        modelTemplateFiles.put("model.mustache", ".js");
-        apiTemplateFiles.put("api.mustache", ".js");
-        embeddedTemplateDir = templateDir = "Javascript-Closure-Angular";
-        apiPackage = "API.Client";
-        modelPackage = "API.Client";
-
         cliOptions.add(new CliOption(CodegenConstants.HIDE_GENERATION_TIMESTAMP, "hides the timestamp when files were generated")
                 .defaultValue(Boolean.TRUE.toString()));
+        cliOptions.add(new CliOption(USE_ES6,
+                "use plain ES6 templates instead - drops Google Closure Compiler annotations")
```

Comment:
Ah sorry @achew22 & @wing328 Probably a misunderstanding on my part - are we looking to only keep the doc blocks as originally provided in the Closure Templates? If so I'll add those in and update.